.github/workflows/integrationtests.yml

```diff
@@ -28,7 +28,23 @@ jobs:
           PROJECT_ROOT: ${{ github.workspace }}
         run: docker-compose up -d
 
-      # Step 4: Test if the server is up
+      # Step 4: Check if Ollama server is up
```

Comment:
Lets remove this file from the pull request
.github/workflows/integrationtests.yml